agent-nodejs-8/Dockerfile

```diff
@@ -16,7 +16,7 @@ COPY contrib/bin/configure-agent /usr/local/bin/configure-agent
 RUN yum install -y centos-release-scl-rh && \
     INSTALL_PKGS="rh-nodejs${NODEJS_VERSION} rh-nodejs${NODEJS_VERSION}-npm rh-nodejs${NODEJS_VERSION}-nodejs-nodemon make gcc-c++" && \
     ln -s /usr/lib/node_modules/nodemon/bin/nodemon.js /usr/bin/nodemon && \
-    yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
+    yum install -y --setopt=protected_multilib=false --setopt=tsflags=nodocs $INSTALL_PKGS && \
```

Comment:
why does nodejs need this? it doesn't install javac right?

Reply:
tried to explain in the description, apologies for lack of clarity ... this was a preemptive decision in case something else popped up in the future ...

but if we should wait until the instances pop up, I'll remove

Reply:
you caught me not reading pr descriptions again. :)

i'd be inclined to wait in case there are other decisions that should be made when we get to that point.